### PR TITLE
Type exposed_in decorator

### PR DESCRIPTION
**Pull request by zou3519**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #141883
* #141880
* __->__ #141894

Test Plan:
- lintrunner

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Ok!

File: torch/_library/custom_ops.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def custom_op(
     mutates_args: Union[str, Iterable[str]],
     device_types: device_types_t = None,
     schema: Optional[str] = None,
-) -> Callable:
+) -> Any:
```
Comment (Skylion007) [inline torch/_library/custom_ops.py:29]: Why remove this valuable typing?

Comment (zou3519) [reply torch/_library/custom_ops.py:29]: It's not correct

File: torch/utils/_exposed_in.py [hunk 2]
```diff
@@ -1,4 +1,9 @@
-# mypy: allow-untyped-defs
+from typing import Callable, TypeVar
+
+
+F = TypeVar("F")
```
[the remark this answers is not in the document]
Comment (zou3519) [reply torch/utils/_exposed_in.py:4]: This file isn't public (the path is torch.utils._exposed_in) so F isn't public

**Pull request closed without merging**